src/domain/repositories/HcvExportRepository.ts

```diff
@@ -0,0 +1,12 @@
+import { Async } from "domain/entities/Async";
+import { HcvSettings } from "domain/entities/HcvSettings";
+import { ProgramEvent } from "domain/entities/ProgramEvent";
+
+export interface HcvExportRepository {
+    saveReport(events: ProgramEvent[], options: HcvExportOptions): Async<void>;
+}
+
+export type HcvExportOptions = {
+    csvPath: string;
```

Comment:
After the discussion on the TD meeting with Jorge, we agreed to keep the repository, with these changes: 1) let's call it HcvReportRepository, 2) csvPath is passed to the data repo constructor, not to the useCase/domain repo methods.